tests/BoundaryTest.cpp

```diff
@@ -89,11 +89,13 @@ TEST_CASE("Boundary unit test", "[Boundary]")
     }
     SECTION("Non-matching length error test")
     {
-        REQUIRE_THROWS_AS(Boundary("[0.0, 1.0, 3.0]", "[0.0, 0.5, 1.0, 1.5]"), std::runtime_error);
+        REQUIRE_THROWS_AS(Boundary("[0.0, 1.0, 3.0]", Json::Value("[0.0, 0.5, 1.0, 1.5]")),
+                          std::runtime_error);
     }
     SECTION("Unordered time error test")
     {
-        REQUIRE_THROWS_AS(Boundary("[0.0, 10.0, 3.0]", "[0.0, 0.5, 1.0]"), std::runtime_error);
+        REQUIRE_THROWS_AS(Boundary("[0.0, 10.0, 3.0]", Json::Value("[0.0, 0.5, 1.0]")),
+                          std::runtime_error);
     }
 }
 TEST_CASE("Traction test for triangle", "[Traction]")
```

Comment:
Are we going to ditch the old value/time and implement your new interface?  If so you'll probably want some help getting this changed across the examples / tests 👍